scss/_grid.scss

```diff
@@ -10,7 +10,7 @@ Mobile First - This applies from 0px to 767px
 
 /***** Row setup *****/
 .row                                            { margin:0 auto; padding:0 20px; display:flex; flex-direction:row; box-sizing:border-box; flex:0 1 auto; flex-wrap:wrap; }
-.row.row--width-fluid                           { width:100%; }
+.row .row--width-fluid                          { width:100%; }
```

Comment:
You still need to declare like you said:
`<div class="row row--width-fluid"></div>`
You only would save the one `.row` in the scss.

But we could just @extend .row--width-fluid with the .row declarations, like so:
```
.row { margin:0 auto; padding:0 20px; display:flex; flex-direction:row; box-sizing:border-box; flex:0 1 auto; flex-wrap:wrap; }
.row--width-fluid { @extend .row; width:100%; }
```
Then we would save the `.row` in html and it would be a bit more BEM.